leader-election-impl/src/main/java/com/palantir/paxos/PaxosAcceptorImpl.java

```diff
@@ -74,8 +74,10 @@ public PaxosPromise prepare(long seq, PaxosProposalId pid) {
 
             // allow for the same propose to be repeated and return the same result.
             if (oldState != null && pid.compareTo(oldState.lastPromisedId) == 0) {
-                leaderLog.debug("Accepted proposal request for seq #{} with ID {} as we've already made a promise with the same pid ({}) " +
-                        "for leader UUID {}", seq, pid.getNumber(), oldState.lastPromisedId, oldState.lastAcceptedValue.getLeaderUUID());
+                String lastAcceptedLeader = (oldState.lastAcceptedValue == null) ? null : oldState.lastAcceptedValue.getLeaderUUID();
```

Comment:
nit: a String like `"(none)"` or `"(null)"` might be better here, rather than `null`.